packages/hub/tasks/notify-customer-payment.ts

```diff
@@ -88,13 +89,20 @@ export default class NotifyCustomerPayment {
       });
     }
 
-    let notifiedAddress = result.merchant.id;
+    let ownerAddress = result.merchant.id;
     let spendAmount = result.spendAmount;
-    let message = `${merchantName} received a payment of §${spendAmount}`;
+    let notificationBody = `${merchantName} received a payment of §${spendAmount}`;
 
-    await this.workerClient.addJob('send-notifications', {
-      notifiedAddress,
-      message,
-    });
+    // TODO: find out which device-address combinations need notifying
+    // and add a job for each (CS-2726)?
```

Comment:
I imagined this happening in `send-notifications` so it doesn’t have to be duplicated in every transaction-to-notification task 🤔

Reply:
I'm not too sure about this either. I think having it here makes it easier to handle partial failures in `send-notifications` where we send some notifications successfully but fail at others.

Reply:
I think the logic should be here - @Aierie good point on separating the retryable worker concerns. I'm currently working on this part in another PR